### ISSUE TITLE
Trying to use buffer-size:  Map properties are not permitted in other rules

### ISSUE THREAD

**Issue by iriberri** [CartoDB/cartodb#2626]
My CartoCSS:
````
Map {
  buffer-size: 256;
}
#spreadsheet_test{
  marker-fill-opacity: 0.9;
  marker-line-color: #FFF;
  marker-line-width: 1.5;
  marker-width: 10;
  marker-fill: #FF6600;

}
````

Happening here but also in a couple more of vizs I tried: https://asdfgh.cartodb.com/u/crb/viz/2665604c-c18e-11e4-a117-0e0c41326911/map

![image](https://cloud.githubusercontent.com/assets/1730320/6492755/101b1a56-c2b6-11e4-800c-14c4e67c2e9a.png)

Not really sure if I'm doing something wrong or if there's a bug here.

**Comment by saleiva** [comment]
Hey @rambo, I think @andrewxhill also experienced this some time ago

**Comment by iriberri** [comment]
I found it sending an answer here: SB/4700702 that I won't be able to send until we know what happens :smile:

**Comment by fdansv** [comment]
Fixed at #2763
